app/models/appointment.rb

```diff
@@ -44,6 +44,10 @@ def created_at
     super || Time.zone.now
   end
 
+  def gmt_or_bst
```

Comment:
Maybe call this`timezone` instead? Also it should return the string literal 'GMT' not 'UTC'. Nobody will know what UTC is.